### PR TITLE
Add file drag over event emitter on enter and leave

### PR DESCRIPTION
**Pull request by AdrianPasalega**
Add an event emitter to be able to modify drop zone css when moving the file over:
(fileOver)="fileOverDropZone($event)"

### CODE REVIEW

**Comment by jkuri** [comment]
Hi @AdrianPasalega, thank you for your PR. That looks alright, but nobody will know about usage of this. Can you please also add this to the README with the example and I will accept it? Thanks.

**Comment by AdrianPasalega** [comment]
Hi @jkuri , yes, you are right :), I add an example, let me know if it is ok.

File: README.md [hunk 1]
```diff
@@ -79,6 +84,17 @@ export class DemoApp {
        [options]="options"
        (onUpload)="handleUpload($event)">
 
+<!-- drag & drop file example-->
+<style>
+  .file-over { border: dotted 3px red; } /* Default class applied to drop zones on over */
+</style>
+<div ng2FileDrop
```
Comment (jkuri) [inline README.md]: typo. directive is named `ngFileDrop` not `ng2FileDrop` so this example would not work :) please fix.

Comment (AdrianPasalega) [inline README.md]: done

**Comment by jkuri** [comment]
Thanks! Merged and published new version to npm.

**Pull request merged**